samples/bluetooth/mesh/silvair_enocean/README.rst

```diff
@@ -0,0 +1,198 @@
+
+After provisioning and configuring the mesh models supported by the sample using the `nRF Mesh mobile app`_, you can control the LEDs on the other (server) development kit(s) from the app.
+
+Provisioning
+============
+
+Provisioning is handled by the :ref:`bt_mesh_dk_prov`.
+It supports four types of out-of-band (OOB) authentication methods, and uses the Hardware Information driver to generate a deterministic UUID to uniquely represent the device.
+
+Models
+======
+
+The following table shows the mesh Silvair EnOcean composition data for this sample:
+
+.. table::
+   :align: center
+
+ +==============================+===================+
+ | Element 1                    | Element 2         |
+ +==============================+===================+
+ | Config Server                | Gen. Level Client |
+ +------------------------------+-------------------+
+ | Health Server                | Gen. OnOff Client |
+ +------------------------------+-------------------+
+ | Gen. Level Client            |                   |
+ +------------------------------+-------------------+
+ | Gen. OnOff Client            |                   |
+ +------------------------------+-------------------+
+ | Silvair EnOcean Proxy Server |                   |
+ +------------------------------+-------------------+
+
+The models are used for the following purposes:
+
+* The Silvair EnOcean Proxy Server instantiates :ref:`bt_mesh_onoff_cli_readme` and :ref:`bt_mesh_lvl_cli_readme` on both elements, where each element is controlled by the buttons on the EnOcean switch.
+* Config Server allows configurator devices to configure the node remotely.
+* Health Server provides ``attention`` callbacks that are used during provisioning to call your attention to the device. These callbacks trigger blinking of the LEDs.
+
+The model handling is implemented in :file:`src/model_handler.c`, which uses the :ref:`dk_buttons_and_leds_readme` library to detect button presses on the development kit.
+
+The response from the target device updates the corresponding LED on the mesh Silvair EnOcean device.
+When the target device is turned on or off, the corresponding LED will turn on or off accordingly.
+If the light level is increased above `BT_MESH_LVL_MIN` on the target device, the corresponding LED will turn on.
+The LED will turn off if the light level is equal to `BT_MESH_LVL_MIN`.
+
+.. _bluetooth_mesh_silvair_enocean_user_interface:
+
+User interface
+**************
+
+Development kit buttons:
+      During the provisioning process, the buttons (1 to 4) can be used for OOB input.
+      Once the provisioning and configuration are completed, the buttons are not used.
+
+EnOcean buttons:
+      Pressed and releasing the button on the EnOcean switch, the button publishes an OnOff message using the configured publication parameters of its model instance, and toggles the LED state on a :ref:`mesh light <bluetooth_mesh_light_lc>` device.
```

Comment:
I still find this sentence a bit strange...
Would it make sense changing to
Pressing and releasing the button?
or
Releasing a pressed button?